Makefile

```diff
@@ -14,6 +14,17 @@ build:
 	# Build with all features
 	cargo build --locked --all-features
 
+# Generates api-server crate from ceramic.yaml OpenAPI spec
+.PHONY: api-server
+api-server: api/ceramic.yaml
```

Comment:
should  `api-server` be two phonys? `generate-api-server` and `check-api-server` so the `all` can check that the generated code is yonger then the `api/ceramic.yaml`